.github/workflows/aws_tfhe_integer_tests.yml

```diff
@@ -26,6 +26,7 @@ on:
 
 jobs:
   should-run:
+    if: github.event_name == 'push' && github.repository == 'zama-ai/tfhe-rs'
```

Comment:
too restrictive on the event_name, it's probably the same condition as the setup instance that is required